src/components/SecondaryPanes/Frames/Group.js

```diff
@@ -20,17 +17,18 @@ import FrameComponent from "./Frame";
 import type { LocalFrame } from "./types";
 import Badge from "../../shared/Badge";
 
-type FrameLocationProps = { frame: LocalFrame };
-function FrameLocation({ frame }: FrameLocationProps) {
+type FrameLocationProps = { frame: LocalFrame, expandedState: boolean };
```

Comment:
let's use `expanded` instead of `expandedState` in order to be more consistent with the rest of the codebase.
```suggestion
type FrameLocationProps = { frame: LocalFrame, expanded: boolean };
```